### DISCUSSION TITLE
How to you get additional user claims?

### DISCUSSION THREAD

**Discussion by Stonebkfly** [oidc-wp/openid-connect-generic#482] [Q&A] [answered]
I want to determine if a user was being impersonated when they login. This information is stored in the AMR sso claim. Ie. AMR = "imp" when a user is being impersonated. How do I get this information in our WordPress plugin?

**Comment by Stonebkfly** [comment, marked as answer]
No sooner did I post the question then I figured out the solution. The "openid-connect-generic-update-user-using-current-claim" action has an input parameter of $user_claim. This is an array of all the claims.

**Reply by timnolte** [reply]
I think probably what's needed is to expand the Wiki with more information and examples such as this. We do have most, if not all, of the hooks & filters documentation there but providing folks with some of these real examples of using the hooks can help others out.
